index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Philemon portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body class="page">
+        <header class="menu">
+            <nav> 
+                <img src="images/normal-button.svg" alt="Menu">
+            </nav>
+        </header>
+        <section>
+           <div class="headline">
+            <h1 class="greetings"> Hey there <br>
+                I'm Philemon <br>
+                <span class="job">I'm a Software Developer</span>
+              </h1>
+              <p class="description"> I can help you build a product, feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
+                don’t hesitate to contact me.</p>
+                <ul class="socials">
+                    <li><a href="#"><img src="images/twitter.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/linkedin.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/medium.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/github.svg" alt="" /></a></li>
+                    <li><a href="#"><img src="images/angellist.svg" alt="" /></a></li>
+                </ul>
+           </div> 
+           <div class="button">
+            <img src="images/vector-button.svg" alt="button">
+          </div>
```

Comment:
- [ ] Kindly make sure to use to a `button` or `a` tag  instead of using the `div` tag.